src/controllers/user/controls/index.js

```diff
@@ -20,7 +21,7 @@ export default function (view) {
 
     view.addEventListener('viewshow', function () {
         view.querySelector('.smoothScrollContainer').classList.toggle('hide', !layoutManager.tv);
-
+        view.querySelector('.enableGamepadContainer').classList.toggle('hide', browser.edgeUwp);
```

Comment:
It would probably be a good idea to extract `browser.edgeUwp` into a `canEnableGamepad` function (in `keyboardNavigation`) or something like that. So we can use it here and in  `keyboardNavigation` to eliminate duplication.